package.json

```diff
@@ -1,50 +1,58 @@
 {
   "name": "ember-router-meta",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
@Mikek2252  patch version?

Reply:
Yeah, i wasnt sure on this because it shouldn't break anything the dependencies are just being bumped. So i'm under the impression that it isn't a breaking change